test/instances/aave/AaveHyperdrive.t.sol

```diff
@@ -110,12 +110,12 @@ contract AaveHyperdriveTest is InstanceTest {
     /// @param _factory The address of the Hyperdrive factory.
     function deployCoordinator(
         address _factory
-    ) internal override returns (address) {
+    ) public override returns (address) {
         vm.startPrank(alice);
```

Comment:
I'm attempting to deploy this contract, then call this function to deploy the coordinator, but getting execution reverted on this line. Are cheatcodes supported when the contract is deployed?